vue-app/src/components/ProjectProfile.vue

```diff
@@ -192,6 +192,11 @@ export default class ProjectProfile extends Vue {
   get addressName(): string {
     return this.ens || this.project.address
   }
+
+  get shouldShowCartInput(): boolean {
```

Comment:
Imported same logic we have in `Project.vue` in order to have the same behaviour in mobile and desktop.